readme.md

```diff
@@ -4,13 +4,12 @@
 
 ![Home view of Decoder Ring](./docs/home.png)
 
-This project is designed to test your ability to build tricky algorithms as well as write unit tests with Mocha & Chai. Before taking on this module, you should be comfortable with the learning objectives listed below. You will not need to make any edits to HTML or CSS for this project.
+This project is designed to test your ability to build complex algorithms as well as write unit tests with Mocha & Chai to test your algorithms. Before taking on this module, you should be comfortable with the learning objectives listed below. You will not need to make any edits to HTML or CSS for this project.
 
 ## Learning Objectives
 
-This project will assess the following learning objectives, in addition to many others:
+This project will assess the following key learning objectives:
```

Comment:
imo, no need to hint at secondary learning objectives if we're not going to list them here...to keep the students focused on the most important goals of the lesson.